content/construction/space_park_way_closure.md

```diff
@@ -35,6 +33,6 @@ Space Park will be divided into two unequal lanes between Shoreline and Santiago
 
 ## Expected Benefits for Santiago Villa
 
-- Space Park Way will be completely repaved.
 - The Inigo extension from La Avenida to Pear to Space Park will create additional access to and from the mobile home park.
-- The infrastructure renovation is likely to stop the annual flooding to the front of the park.
\ No newline at end of file
+- The infrastructure renovation is likely to stop the seasonal flooding to the front of the park.
+- The eventual realignment of Plymouth with Space Park should add a stoplight at the corner of Shoreline and Space Park.
```

Comment:
Can we get a roundabout instead?

Reply:
The benefits I listed are just collateral improvement. Mentioning them that might make this project seem like a little less of a pain in the ass to the folks who have to figure out where to put their cars. Maybe.  
Sobrato is probably not going to do us any favors, but hopefully we can get our park management on board for the gate joining El Centro with Pear. That's the only real benefit they initially offered us. But as I recall the area they are constructing now will look kind of like a roundabout when it's done. Maybe that's what you're referring to, and if so, I concur.